src/Command/ElFinderInstallerCommand.php

```diff
@@ -22,21 +22,19 @@ final class ElFinderInstallerCommand extends Command
 
     private const ELFINDER_IMG_DIR    = 'vendor/studio-42/elfinder/img';
 
-    protected $fileSystem;
+    protected static $defaultName = 'elfinder:install';
 
-    protected $parameterBag;
 
-    public function __construct(Filesystem $filesystem, ParameterBagInterface $parameterBag)
-    {
-        $this->fileSystem   = $filesystem;
-        $this->parameterBag = $parameterBag;
+    public function __construct(
+        protected Filesystem $filesystem,
+        protected ParameterBagInterface $parameterBag
+    ) {
         parent::__construct();
     }
 
     protected function configure(): void
     {
         $this
-            ->setName('elfinder:install')
             ->setDescription('Copies elfinder assets to public directory')
```

Comment:
You can also set the description via `protected static $defaultDescription = 'Copies elfinder assets to public directory';`